Algorithm.CSharp/BasicTemplateIntrinioEconomicData.cs

```diff
@@ -61,6 +62,11 @@ public override void Initialize()
             // Set your Intrinino user and password.
             IntrinioConfig.SetUserAndPassword(_user, _password);
 
+            // Set Intrinio config to make 1 call each minute, default is 1 call each 5 seconds.
+            // (1 call each minute is the free account limit for historical_data endpoint)
+            IntrinioConfig.RateGate = new RateGate(1, TimeSpan.FromMinutes(1));
```

Comment:
Maybe we can do this via `IntrinoConfig.SetUserAndPassword` and have it accept the rate limit parameters, defaulting to the free account?

Reply:
Great idea! 
Done.